text/chapter10.md

```diff
@@ -1009,14 +1036,15 @@ mkAddressBookApp =
 ```
 
 As a finishing touch, we'll improve the quality of our error messages by appending to the `String` of each `Left` value with `lmap`.
+
 ```hs
 processItem :: Json -> Either String Person
 processItem item = do
   jsonString <- lmap ("No string in local storage: " <> _) $ decodeJson item
-  j <- lmap ("Cannot parse JSON string: " <> _) $ jsonParser jsonString
-  (p :: Person) <- lmap ("Cannot decode Person: " <> _) $ decodeJson j
-  pure p
+  j          <- lmap ("Cannot parse JSON string: "   <> _) $ jsonParser jsonString
+  lmap               ("Cannot decode Person: "       <> _) $ decodeJson j
 ```
+
```

Comment:
Code anchors would be good for these sections too, but that can be tackled in another PR.

Reply:
Yeah, I don't remember why I skipped this one.